test/integration/targets/elb_target/playbooks/roles/elb_target/tasks/main.yml

```diff
@@ -102,6 +102,19 @@
           Description: "Created by {{ resource_prefix }}"
         <<: *aws_connection_info
 
+    - name: set up testing target group (type=instance) with UDP protocol
+      elb_target_group:
+        name: "{{ tg_tcpudp_name }}"
```

Comment:
this variable will need to be created for this test suite in test/integration/targets/elb_target/playbooks/roles/elb_target/defaults/main.yml

Reply:
oh man, sorry :D